telosnetwork/telos-wallet
# #365 | NFT preview component

# Fixes #365

## Description
This PR adds an NFT preview component, for use on the inventory page

## Test scenarios
- go to
- log in
- click the new NFT link
    - you should see a barebones page with NFT previews
    - the various types of NFT should display according to the acceptance criteria
- change the size of your screen progressively down to mobile
    - the page should be responsive
- click on an NFT
    - you should be directed to the detail page with the correct URL params (contract and ID)
-  go back
- click on the external link under the nft
- you should be taken to the correct teloscan page 

## Checklist:
